### PR TITLE
add .dockerignore.

### PR DESCRIPTION
**Pull request by Camiloarceg**
Hi guys I have this option for it to be our .dockerignore
I have retrieved from: https://github.com/GoogleCloudPlatform/getting-started-python/blob/main/optional-kubernetes-engine/.dockerignore

which is part of a getting-started-python course offered by Google Cloud Platform.
In my opinion, is very complete and makes sense with the unit test library and code coverage that we are using, tell me what you think.

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+pip-delete-this-directory.txt
+.tox
+.coverage
+.coverage.*
+.cache
+nosetests.xml
+coverage.xml
+*.cover
+*.log
+.git
```
Comment (aolguin89) [inline .dockerignore:17]: ¿Maybe .git* or /.git* ? (to include .gitignore)

**Review by aolguin89 — CHANGES_REQUESTED** [review]
Hello Camilio, it looks very good to me, for our particular case it seems to me that the "bank_insomnia-v1.json" file might not be strictly necessary, we could add it

File: .dockerignore (hunk 1 above)
Comment (calebespinoza) [reply .dockerignore:17]: Sure, as we saw in class, every file or folder which are not needed to run the app should be ignored, those include `.git` folder, and even, `Dockerfile`, `.dockerignore`, etc.

**Comment by Camiloarceg** [comment]
Closing This PR, Was used as base for already merged PR#16

**Pull request closed without merging**